modules/manage/pages/kubernetes/k-scale-redpanda.adoc

```diff
@@ -282,3 +293,9 @@ kubectl get persistentvolumeclaim --namespace <namespace>
 kubectl get persistentvolume --namespace <namespace>
 ----
 
+After the Nodewatcher controller has finished, xref:manage:kubernetes/k-decommission-brokers.adoc[decommission the broker] that was removed from the node. This is necessary to prevent quorum loss and ensure cluster stability.
+
+NOTE: Make sure to use the `--force` flag when decommissioning the broker with xref:reference:rpk/rpk-redpanda/rpk-redpanda-admin-brokers-decommission.adoc[`rpk redpanda admin brokers decommission`]. This flag is required when the broker is no longer running.
```

Comment:
I don't know the ins and outs of `--force`, could there be other ramifications here? For example if there's a topic with RF=1 that had a partition on the lost node does this opt into data loss or is that a different ball game? Ignore more if you know this is safe :)

Reply:
Just chiming in, `--force` was added to avoid the safety checks performed in older clusters:

> For safety on v22.x clusters, this command will not run if the requested 
broker is in maintenance mode. As of v23.x, Redpanda supports 
decommissioning a node that is currently in maintenance mode. If you are on 
a v22.x cluster and need to bypass the maintenance mode check (perhaps your 
cluster is unreachable), use the hidden --force flag.

More info here: https://github.com/redpanda-data/redpanda/commit/85c05713a8bcf148a5fd2c05c598748ee597dc92

Reply:
Thanks Rogger. So that flag isn’t required for versions after 22.x?

Reply:
It shouldn't be. But there are scenarios (like a cluster is unreachable) where you still want to use this.

Cc: @daisukebe since you know more about the edge cases when using this command :+1:

Reply:
Since we already document this flag, we can add follow-up info about edge cases/ramifications. I'll merge this one for now.

Reply:
> This flag is required when the broker is no longer running.

This is still true in the latest release of Redpanda. When a broker is dead and we want to decommission it, the force flag is required. Read the Nodewatcher doc and it seems like the case.